CONTRIBUTING.md

```diff
@@ -8,48 +8,48 @@ Welcome to [TheAlgorithms/C-Plus-Plus](https://github.com/TheAlgorithms/C-Plus-P
 
 ### Contributor
 
-We are very happy that you consider implementing algorithms and data structures for others! This repository is referred to and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
+We are happy that you consider implementing algorithms and data structures for others! This repository is referred to and used by learners from around the globe. Being one of our contributors, you agree and confirm that:
 
 - You did your own work.
   - No plagiarism allowed. Any plagiarized work will not be merged.
 - Your work will be distributed under [MIT License](License) once your pull request has been merged.
-- You submitted work fulfils or mostly fulfils our styles and standards.
+- Your submitted work fulfills or mostly fulfills our styles and standards.
 
-**New implementation** New implementation are welcome!
+**New implementation** New implementations are welcome!
 
 **Improving comments** and **adding tests** to existing algorithms are much appreciated.
```

Comment:
- You can add new algorithms or data structures which are not present in repository or can improve the old implementations (documentation, improving test cases, removing bugs or in any other resonable sense)